cpp/src/Ice/UndefSysMacros.h

```diff
@@ -1,8 +1,6 @@
 // Copyright (c) ZeroC, Inc.
-
-/*! \file
-    \hideincludedbygraph
-*/
```

Comment:
This is another special kind of block-comment that your script should leave alone.
Matching for `/**` instead of just `/*` should also avoid these though : v3